static/css/theme-activestate.css

```diff
@@ -64,10 +143,10 @@ a:hover {
     content: "";
     height: 1px;
     width: 0%;
-    -webkit-transition: width 0.5s ease;
-    -moz-transition: width 0.5s ease;
+    -webkit-transition: width 0s ease;
+    -moz-transition: width 0s ease;
     -ms-transition: width 0.5s ease;
-    transition: width 0.5s ease;
+    transition: 0
```

Comment:
Can you double-check if transition 0 is valid? Might be something like `transition: none` and also, we should probably match the rule for all of the prefixed ones as well.